src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/ClientsApiServiceImpl.scala

```diff
@@ -1,5 +1,6 @@
 package it.pagopa.interop.backendforfrontend.api.impl
 
+import cats.implicits._
```

Comment:
```suggestion
import cats.syntax.all._
```

`cats.implicits._` is deprecated: https://typelevel.org/cats/imports.html